lib/src/commands.dart

```diff
@@ -421,6 +479,22 @@ class CommandsPluginImpl extends BasePlugin with GroupMixin implements CommandsP
 
           builder.registerHandler((interaction) => processUserInteraction(interaction, command));
 
+          builders.add(builder);
+        } else if (command is MessageCommand) {
+          SlashCommandBuilder builder = SlashCommandBuilder(
+            command.name,
+            null,
+            [],
+            defaultPermissions: defaultPermission,
+            permissions: List.of(
+              permissions.where((permission) => permission.id != Snowflake.zero()),
```

Comment:
`Snowflake.zero()` here would allocate new object each iteration over permissions, depending on case it could be very inefficient